model.py

```diff
@@ -132,7 +132,7 @@ def fit(self, x, f, v=None, a=[], b=[], s=[],t=[], c=[], v0=None, a0=[], b0=[],
             self.show(p, x, f, v=v, **kwargs)
         return p, e_p
 
-    def show(self, p, x, y, v=None, res=True, x2=None, dx=None):
+    def show(self, p, x, y, v=None, res=True, x2=None, dx=None, notell=None):
```

Comment:
add `notell` to the docstring.

Reply:
You mean, you try to figure out was the notell plotting option does? (just to prevent more misunderstanding)

Reply:
Yes. See also the comment below.

Reply:
Does my answer from below answer the question from here?